### PR TITLE
#160583955 set up react router

### PR DESCRIPTION
**Pull request by balaakagordon**
#### What does this PR do?
Set up routing for navigation between pages
#### Description of Task to be completed?
- enable navigation, in the browser, to the following endpoints:
`'/'`
`'/login'`
- display error message for other endpoints
#### How should this be manually tested?
After pulling the branch:
- update dependencies using
`npm install`
- run the application using
`npm start`
- navigate to the aforementioned endpoints
- try visiting an endpoint that does not exist yet
#### What are the relevant pivotal tracker stories?
#160583955 - Setup react router version 4 (React-router-dom)
https://www.pivotaltracker.com/n/projects/2198584/stories/160583955
#### Screenshots (if appropriate)
Landing page
image
-------------------------------------------------------------------------------------------------

Login page
image
-------------------------------------------------------------------------------------------------

Page doesn't exist
image
-------------------------------------------------------------------------------------------------

### CODE REVIEW

**Review by kenneth051 — CHANGES_REQUESTED** [review]
Nice work Gordon the routes do work. However, when I add a slash and some words on the login route, it still navigates to the login page. Try adding "exact" to all the routes and we see if it will catch the route errors. I also haven't seen any tests written for the feature.

**Comment by balaakagordon** [comment]
Thanks for the feedback, Kenneth. I have implemented the changes

**Review by kenneth051 — APPROVED** [review]

File: .eslintrc.json [hunk 1]
```diff
@@ -1,3 +1,10 @@
 {
+<<<<<<< HEAD
```
Comment (kenneth051) [inline .eslintrc.json]: @balaakagordon, please resolve these conflicts.

File: src/App.js [hunk 2]
```diff
@@ -1,12 +1,26 @@
...
+
 
-class App extends Component {
-  render() {
-    return (
-      <h1>  Welcome to Authors-Haven</h1>
-    );
-  }
-}
+const App = () => (
```
Comment (kenneth051) [inline src/App.js]: @balaakagordon Since we will have to use state, could you please revert this back to a class-based component.

**Comment by kenneth051** [comment]
LGTM

**Review by j0flintking02 — APPROVED** [review]
LGTM

**Pull request merged**